event/src/bytes.rs

```diff
@@ -1,14 +1,24 @@
 //! Thin wrapper on a Vec<u8> that correctly serializes and deserializes as bytes as opposed to a
 //! sequence of u8 integers.
+use std::fmt::Debug;
+
 use serde::{
     de::{self, Visitor},
     Deserialize, Serialize,
 };
 
 /// Sequence of byte values.
-#[derive(Clone, PartialEq, Eq, Default, Debug)]
+#[derive(Clone, PartialEq, Eq, Default)]
 pub struct Bytes(Vec<u8>);
 
+impl Debug for Bytes {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_tuple("Bytes")
+            .field(&multibase::encode(multibase::Base::Base64Url, &self.0))
```

Comment:
Why did we pick b64 for this?
I find when debugging hex is easier to look at what the bytes are doing as bytes don't span chars. That said the b64 is shorter for use in expect tests.

Reply:
It was arbitrary however I wanted it to have the multibase prefix instead of raw hex since many tools expect multibase (cpk being one). I'll change to base 16 lower. Thanks for the call out.